src/Blockcore/Blockcore.csproj

```diff
@@ -27,6 +27,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="5.5.1" />
```

Comment:
IS this required in the core lib or can it go to the nodes  or dedicated feature?

Reply:
It is needed to add tags, but the tags was not shown in the UI anyway, so I'll remove both the tags and this package.